hack/build/docker/builder/fedora.repo

```diff
@@ -1,4 +1,4 @@
-[fedora-base-fc30]
+[fedora-base-fc29]
```

Comment:
Need to do this for automation/check-patch.yumrepos as well. That is a symlink to all the other fedora.repos that are using in the dockerfiles.

Reply:
I have that here no?  https://github.com/kubevirt/containerized-data-importer/pull/915/commits/52ad4935b02fb2a3f41d8aea41ce06cb4ffb0394#diff-42d85b9ddb8c0d2cec8713a2713f107cR1  Is there another hidden tgem